onthequad/onthequad/Event.py

```diff
@@ -0,0 +1,16 @@
+import uuid
+class Events:
+    def __init__(self, name, location, date, starttime, endtime, category, creator, attendees, availablespots,
+                 coordinates, public):
+        self.id = uuid.uuid1()
+        self.name = name
+        self.location = location
+        self.date = date
+        self.starttime = starttime
+        self.endtime = endtime
+        self.category = category
+        self.creator = creator
+        self.attendees = attendees
+        self.availablespots = availablespots
+        self.coordinates = coordinates
+        self.public = public
```

Comment:
Should leave a blank line at the end.

Reply:
Git doesnt show the blank lines, but does want you to have them. I'm just as confused as you. Sent an example of what git does when it wants a blank line at end of file.